lms/djangoapps/verify_student/tests/test_services.py

```diff
@@ -31,89 +31,101 @@ def setUp(self):
             min_price=100,
         )
         self.item = ItemFactory.create(parent=course, category='chapter', display_name='Test Section')
+        self.final_checkpoint_location = u'i4x://{org}/{course}/edx-reverification-block/final_term_uuid'.format(
+            org=self.course_key.org, course=self.course_key.course
+        )
 
-    @ddt.data("final_term", "mid_term")
+    @ddt.data('final_term', 'mid_term')
```

Comment:
Nitpick: These checkpoint names would be more correct as `'final'` and `'midterm'`.